07-allow-traffic-from-some-pods-in-another-namespace.md

```diff
@@ -93,6 +93,8 @@ If you don't see a command prompt, try pressing enter.
 ...
 (traffic allowed)
 ```
+## remarks
+The above mentioned scenario will only work if the namespace named other has label team=operations because the condition mentioned in our network policy is a AND condition which means both the condition must be true for the network policy to allow ingress traffic
```

Comment:
```suggestion
The above mentioned scenario will only work if the namespace named `other` has label team=operations because the condition mentioned in our network policy is a AND condition which means both conditions must be true for the network policy to allow ingress traffic.
```